api/address.proto

```diff
@@ -9,10 +9,11 @@ message Address {
       TCP = 0;
     }
     Protocol protocol = 1;
-    // For listeners, bind to 0.0.0.0 or :: to support the use of prefix/suffix
-    // matching in FilterChainMatch. For clusters, an address may be either an
-    // IP or hostname to be resolved via DNS.
-    google.protobuf.StringValue address = 2;
+    // For listeners, an empty address implies a bind to 0.0.0.0 or ::. This
```

Comment:
I can't give a full example right now, but I can easily imagine a case where someone wants to bind to a specific local address, say, 1.2.3.4:80 (maybe because something else is binding to 2.3.4.5:80), but then via iptables and USE_ORIGINAL_DST, actually have complex matching rules that go beyond 1.2.3.4. 

I was just pointing out that the matching functionality is not only useful for 0.0.0.0 (though probably is most of the time).